src/OrchardCore/OrchardCore.ContentManagement/Records/Migrations.cs

```diff
@@ -82,7 +82,7 @@ public async Task<int> CreateAsync()
         public async Task<int> UpdateFrom1Async()
         {
             await SchemaBuilder.AlterIndexTableAsync<ContentItemIndex>(table => table
-                .AddColumn<string>("ContentItemVersionId", c => c.WithLength(26))
+                .AddColumn<string>("ContentItemVersionId", column => column.NotNull().WithLength(26))
```

Comment:
You can't add columns with notull constraints here, just when creating the table, because it means an existing site doesn't have the column, and it will fail.

Reply:
True from an Upgrade it could potentially fail.